diesel/src/query_dsl/belonging_to_dsl.rs

```diff
@@ -1,5 +1,46 @@
+/// Loads record(s) by a directional association with (an)other record(s).
+///
+/// # Example
+///
+/// ```rust
+/// // You need to have the table definitions from `infer_schema!` or `table!` in scope to
+/// // derive Identifiable.
```

Comment:
Do we need to restate this here? Is it stated in the docs for `Identifiable`? Do you think this line is confusing without seeing the `use` lines?

Reply:
This example is copy pasta from the associations mod. This didn't jump out at me due to rendering issue.